Libraries/Notification/NotificationAndroid.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
```

Comment:
Oh, actually this file should probably go away since you have a NotificationAndroid under Libraries/Notification and also one below under Libraries/NotificationAndroid